README.md

```diff
@@ -114,6 +114,9 @@ $token = $session->generateToken(array(
 
 ## Working with Archives
 
+**Important:** You can only archive sessions that use the OpenTok Media Router
```

Comment:
Yes, we have had comments that this requirement (using routed sessions for archiving) is not made clear enough.